utils/net.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package utils
+
+import (
+	"net"
+)
+
+func GetOutboundIP() (net.IP, error) {
```

Comment:
💯  for extracting it to pure function. However I recommend leaving it at ip package, because its related to ip right? "util" packages usually mean that we don't know where to put the code and sounds artificial (ofcourse there are cases for them). Anyway I did almost the same on my branch - just renamed function to GetOutbound() since it's in API package. I think:
```go
ip.GetOutbound()
```
looks more readable?

Reply:
`ip.GetOutbound()` is perfect, moved it.

Reply:
One less merge conflict to worry about from my side :)